blocks/library/gallery/gallery-image.js

```diff
@@ -5,7 +5,7 @@ export default function GalleryImage( props ) {
 		case 'media':
 			href = props.img.url;
 			break;
-		case 'attacment':
+		case 'attachment':
```

Comment:
i always misspell that word too - funny that i didn't catch that when reviewing. 😄